bazel/repositories.bzl

```diff
@@ -116,6 +116,10 @@ def envoy_api_deps(skip_targets):
         name = "envoy_eds",
         actual = "@envoy_api//api:eds",
     )
+    native.bind(
```

Comment:
Since googleapis is loaded from envoy-api, so I added here, let me know if this should go somewhere else. @htuch